src/material-examples/example-module.ts

```diff
@@ -715,6 +716,12 @@ export const EXAMPLE_COMPONENTS = {
     additionalFiles: null,
     selectorName: null
   },
+  'table-selection': {
```

Comment:
I assume this was generated and not typed manually? We should probably remove this file from Git even.

Reply:
Yeah that was just the `gulp` task that generates that file.